### DISCUSSION TITLE
Deployment Architecture

### DISCUSSION THREAD

**Discussion by jozb09** [Security-Onion-Solutions/securityonion#8857] [Unsupported Versions]
Hello Team,

I want to understand from your experience how I can have production with the DR side on my deployment. 

NOTE:
I do have like 150+ servers
Network nodes (Firewall, ISE, ASA, VPN)
Workstation around 300+

I want to go with Distributed But include the DR side. Let me hear about your experience on this, please

Having DR ensuring me to have a backup on the other side.

**Comment by reyesj2** [comment]
Just depends on what level of network connectivity you have to your DR site. Is it used as HA? Strictly a failover site? Is it just for data backup / recovery?
Are you willing to add forward nodes to your DR site and enable network connectivity back and forth from your production site manger node < - > DR forward nodes on all the ports listed in the [docs](https://docs.securityonion.net/en/2.3/firewall.html?highlight=port#node-communication) ?

**Comment by jozb09** [comment]
@reyesj2 is it possible to have HA like a failover?

**Comment by jozb09** [comment]
Help me understand how many servers I should have to have the full setup of this.
Because on the description looks like two nodes which will be forwards and two nodes as search nodes and one manager. So, is that on the Prod environment? should it be the same on the DR?

**Comment by reyesj2** [comment]
I am not aware of having two manager nodes for the purpose of HA.

**Reply by OzzyKampha** [reply]
It is not designed for having two master nodes in one deployment. Thats why the reciver node was added. It has the function to allow  the other nodes to send logs to the search nodes, even if the master node goes down.

If you set up a reciver node, it will work together with the master Active/active.

So to achieve HA on the master node. Its  recommended to deploy the master node in i vm cluster, and using HA fuction from that

[docs - reciver node](https://docs.securityonion.net/en/2.3/architecture.html)

**Comment by reyesj2** [comment]
Your amount of forward nodes and search nodes is dependent on how much data you're ingesting. I mentioned having similar setups on both your prod net and your DR net IF you are authorized to allow communication from your DR network & Prod network to ONE manager node. In that case you could run a forward node & search node on your prod site and just a forward node on your DR site to the manager.

**Reply by jozb09** [reply]
So, no search node on the DR just forward??

**Comment by jozb09** [comment]
So, no search node on the DR. it is just a forward node?

**Comment by dougburks** [comment]
The simplest option would be to simply add a forward node to the DR network that would leverage your existing manager and search node(s) in the PROD network. However, depending on your policies and goals, you may need a more complex option like creating a totally independent deployment for the DR network. It all depends on your policies and goals.
